selene-lib/tests/lints/roblox_incorrect_roact_usage/roblox_incorrect_roact_usage.stderr

```diff
@@ -28,9 +28,19 @@ error[roblox_incorrect_roact_usage]: `ThisDoesntExist` is not a valid class
 27 │ e("ThisDoesntExist")
    │   ^^^^^^^^^^^^^^^^^
 
-error[roblox_incorrect_roact_usage]: `Name` is assigned through the element's key for host components
-   ┌─ roblox_incorrect_roact_usage.lua:30:5
+error[roblox_incorrect_roact_usage]: `Name` is assigned through the element's key for Roblox instances
+   ┌─ roblox_incorrect_roact_usage.lua:38:5
    │
-30 │     Name = "Should not be passed",
+38 │     Name = "Should not be passed",
    │     ^^^^
+   │
+   = try: ["Should not be passed"] = e(...)
```

Comment:
Can we add a case too for a name that can be passed in as just `Whatever = e`? We shouldn't be suggesting `[]` unless we need to.